src/server.rs

```diff
@@ -237,7 +237,7 @@ impl<T: Read + Write + ScmSocket> ClientConnection<T> {
 ///
 /// // Server loop processing requests.
 /// loop {
-///     for request in server.requests().unwrap() {
+///     for request in server.requests(None).unwrap() {
```

Comment:
Unrelated to the actual line in `server.rs`, but here we can define a default max size for requests, which we'll pass on to `HttpConnection`. The other option would be to define it in `connection.rs`, but it doesn't matter much as we can expose it in `mod.rs` nicely anyway.

Reply:
If we define it in server I don't think we need to also have it in connection file